### PR TITLE
Fixin typos and nulls

### PR DESCRIPTION
**Pull request by assellalou**
more fixing 💜

image

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -1955,8 +1955,8 @@
         "region_id": 2,
         "names": {
           "ar": null,
```
Comment (mehdibo) [inline all.json]: It would be very great if you add the Arabic translation to this file too

Comment (assellalou) [reply all.json]: ima be fixin all the null stuff cz there r so many lol

File: all.json [hunk 2]
```diff
@@ -1954,9 +1954,9 @@
       {
         "region_id": 2,
         "names": {
-          "ar": null,
-          "en": "Sidi Slimane Echcharaa",
-          "fr": "Sidi Slimane Echcharaa"
+          "ar": "سيدي سليمان الشراعة",
```
Comment (assellalou) [inline all.json:1957]: it's all good for this one I think

**Review by mehdibo — APPROVED** [review]
All good, great work!

**Pull request merged**